assets/src/design-system/components/menu/menu.js

```diff
@@ -47,22 +49,27 @@ import { EmptyList, ListGroupings } from './list';
  *
  */
 
-const Menu = ({
-  dropDownHeight,
-  emptyText,
-  menuStylesOverride,
-  hasMenuRole,
-  isRTL,
-  options = [],
-  listId,
-  onMenuItemClick,
-  onDismissMenu,
-  renderItem,
-  activeValue,
-  menuAriaLabel,
-  parentId,
-}) => {
-  const listRef = useRef();
+const Menu = (
+  {
+    dropDownHeight,
+    emptyText,
+    menuStylesOverride,
+    hasMenuRole,
+    handleReturnToParent,
+    isMenuFocused = true,
+    isRTL,
+    options = [],
+    listId,
+    onMenuItemClick,
+    onDismissMenu,
+    renderItem,
+    activeValue,
+    menuAriaLabel,
+    parentId,
+  },
+  ref
+) => {
+  const listRef = ref;
```

Comment:
So i did start with the ref coming in and not getting assigned to a variable,  but it actually makes the menu un-navigable by a keyboard. It needs to get assigned here like this for the items to be focusable.  Otherwise this happens. 
![ref goes missing](https://user-images.githubusercontent.com/10720454/106960919-11b71380-66fa-11eb-8660-a63b5e8ce4d6.gif)

Reply:
I just tried renaming the `ref` on line 70 to `listRef` and it worked locally for me 🤔